test_package/example.ll

```diff
@@ -0,0 +1,170 @@
+; ModuleID = '/home/lucas/git/itst-develop/02_sa/intellisec-sast/analysis/test/resources/llvm_test_code/two_analyses_approach/ta_test_inter_true_positive.c'
+source_filename = "/home/lucas/git/itst-develop/02_sa/intellisec-sast/analysis/test/resources/llvm_test_code/two_analyses_approach/ta_test_inter_true_positive.c"
```

Comment:
Do you want to shorten these absolute paths?

Reply:
shortened